tests/foreman/longrun/test_inc_updates.py

```diff
@@ -66,19 +66,19 @@ def qe_lce(module_entitlement_manifest_org, dev_lce):
 
 
 @pytest.fixture(scope='module')
-def rhel7_sat6tools_repo(module_entitlement_manifest_org, module_target_sat):
+def sat_client_repo(module_entitlement_manifest_org, module_target_sat):
     """Enable Sat tools repository"""
-    rhel7_sat6tools_repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
+    sat_client_repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
         basearch=DEFAULT_ARCHITECTURE,
         org_id=module_entitlement_manifest_org.id,
         product=PRDS['rhel'],
-        repo=REPOS['rhst7']['name'],
-        reposet=REPOSET['rhst7'],
+        repo=REPOS['rhsclient7']['name'],
```

Comment:
Would it make more sense to test rhel8 or rhel9 instead? Non-blocking just throwing ideas:)